open_prices/api/proofs/views.py

```diff
@@ -94,3 +97,21 @@ def upload(self, request: Request) -> Response:
         proof = serializer.save(owner=self.request.user.user_id, source=source)
         # return full proof
         return Response(ProofFullSerializer(proof).data, status=status.HTTP_201_CREATED)
+
+    @extend_schema(request=ProofProcessWithGeminiSerializer)
+    @action(
+        detail=False,
+        methods=["POST"],
+        url_path="process_with_gemini",
+        parser_classes=[MultiPartParser],
+    )
+    def process_with_gemini(self, request: Request) -> Response:
+        if not request.data.get("files"):
```

Comment:
if already set as required in the Serializer, then its maybe not needed here ?

Reply:
Without it, it seems that requests without the files param result in a 200.

I still removed it, since nothing seems to break without it anyway, it just replies with an empty list.